index.html

```diff
@@ -3,10 +3,63 @@
       <head>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
         <link rel="stylesheet" href="styles/main.css">
-        <title>JT</title>
+        <title>JT's Portfolio</title>
       </head>
 <body>
-    <header> <h1>Jud&aacute; Teixeira</h1> </header>
+  
+  <header>
+    <div>
+      <a href="#" class="logo">
+        <img src="images/banner.png" alt="Jt-logo">
+      </a>
+    </div>
+    
+
+    <div>
+      <nav>
+        <a href="#" class = "menu">
+        <img src="images/menu.png" alt="Menu">
+        </a>
+     </nav>
+    </div>
+  </header>
+  
+  <div class="main">
+  
+  <div class="content">
+
+    <h1>I'm Jud&aacute; <br> Glad to see <br> you!</h1>
```

Comment:
- [ ] (Optional) One of the major requirements states that you should wrap the whole headline section in a `section` tag. I think it would be nice to adhere to this project requirement for semantic reasons. 👍